tests/Query/Functions/Sqlite/JsonGroupObject.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Scienta\DoctrineJsonFunctions\Tests\Query\Functions\Sqlite;
+
+use Scienta\DoctrineJsonFunctions\Tests\Query\SqliteTestCase;
+use Doctrine\ORM\Query\Expr;
+
+class JsonGroupObjectTest extends SqliteTestCase
+{
+    public function testJsonGroupObject()
+    {
+        $this->assertDqlProducesSql(
+            "SELECT JSON_GROUP_OBJECT('a', 1) FROM Scienta\DoctrineJsonFunctions\Tests\Entities\Blank b",
+            "SELECT JSON_GROUP_OBJECT('a', 1) AS sclr_0 FROM Blank b0_"
+        );
+    }
+
+    public function testJsonGroupObjectMissingParameter()
```

Comment:
When I renamed this file to `JsonGroupObjectTest.php`, this test case failed. Could you fix this test? I don't know what the intention of this test is, because it must fail? But this is not asserted...